### DISCUSSION TITLE
Treinaweb ou Alura? Qual o melhor?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2087] [Discussão]
Boa noite! Primeiro gostaria de me desculpar por estar fazendo essa pergunta nessa parte do forum, eu tentei achar outro lugar para postar mas não encontrei.. (É a minha primeira vez postando nesse site..) Qualquer coisa, se o adm quiser eu removo a postagem.
Bem, eu estou em duvida entre a Treinaweb e a Alura. Eu estou querendo cursar redes e aprender sobre banco de dados.. Vocês poderiam me dizer o que acham de cada curso, e se já estudaram em alguma dessas escolas, poderiam me dizer o que acharam? obrigado!!

Redes:
https://www.alura.com.br/formacao-redes
https://www.treinaweb.com.br/formacao/administrador-de-redes

Banco de dados:
https://www.treinaweb.com.br/formacao/analista-de-banco-de-dados
https://www.alura.com.br/cursos-online-data-science/sql

**Comment by jpferreiradev** [comment]
Acredito que as duas são ótimas para iniciar, tudo vai depender da sua preferencia e bolso. Estudo na Alura já faz uns 2 anos, entre indas e vindas, e para mim a plataforma deles é muito boa, elas já tem um bom tempo no mercado e são otímas para quem está começando ou até para quem já está na área, qualquer coisa veja a emenda do curso de redes e banco de dados de cada uma, só para ter um critério a mais e bons estudos....

**Reply by ghost** [reply]
Oi, obrigado pela resposta! Eu to achando a Alura muito básica nos cursos. Tipo, tenho a impressão que ela só da uma leve introdução no que é algumas coisas e não avança muito..

**Reply by jpferreiradev** [reply]
Eu achei isso no ínicio também, vejo que aos poucos ela está atualizando os cursos, mas para quem está começando eu vejo com um boa entrada, ela não é perfeita, e é caro a se comparar com outros cursos, mas ai vai do gosto e do bolso...
